src/Core/Worker.php

```diff
@@ -1208,27 +1234,30 @@ public static function defer()
 		$id = $queue['id'];
 		$priority = $queue['priority'];
 
-		if ($retrial > 14) {
-			Logger::log('Id ' . $id . ' had been tried 14 times. We stop now.', Logger::DEBUG);
+		$max_level = 15;
```

Comment:
Please introduce this value as a `const MAX_RETRIEVAL_LEVEL = 15;`
So it's possible to adjust such values without grooming the code :-)